invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml

```diff
@@ -250,6 +264,7 @@
     type: publication
   title:
     en: Taxonomic treatment
+    de: Steuerliche Abhandlung
```

Comment:
`Taxonomische Behandlung` (auf keinen Fall "steuerlich")

Reply:
Thanks for checking @Herrner